source/_UIAHandler.py

```diff
@@ -183,6 +183,25 @@ def MTAThreadFunc(self):
 				isUIA8=True
 			except (COMError,WindowsError,NameError):
 				self.clientObject=CoCreateInstance(CUIAutomation._reg_clsid_,interface=IUIAutomation,clsctx=CLSCTX_INPROC_SERVER)
+			# #7345: Instruct UIA to never map MSAA winEvents to UIA propertyChange events.

```

Comment:
I"m still an advocate of making this configurable with a hidden config parameter, i.e. `config.conf['UIA']['unregisterProxyWinEvents']`